clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/LeaderFollowerStoreIngestionTask.java

```diff
@@ -188,7 +188,8 @@ public class LeaderFollowerStoreIngestionTask extends StoreIngestionTask {
 
   private final AtomicLong lastSendIngestionHeartbeatTimestamp = new AtomicLong(0);
 
-  protected final int maxRecordSizeBytes; // TODO: move into VeniceWriter when nearline jobs enforce max record size
```

Comment:
If I understand, this is the max record size for batch data, can you explain a bit why we can remove it here?

Reply:
This max record size variable is only for nearline data. The one pertaining to batch data is set by the `DataWriterJob` onto `VeniceWriter` on the VPJ side ([link](https://github.com/linkedin/venice/blob/aea19bfa64b98ed9c34e23a2ef454c17f97b95a3/clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/task/datawriter/AbstractPartitionWriter.java#L358)).

Reply:
Got it and thanks for the explanation.